toolkit/tools/pkg/imagecustomizerlib/testdata/multikernel-azl2.yaml

```diff
@@ -0,0 +1,10 @@
+os:
+  packages:
+    install:
+    - kernel-5.15.167.1-2.cm2
```

Comment:
curious, does grub regenerates grub.cfg when new kernels are installed? or do we trigger that in prism?

Reply:
Both. The kernel package install will trigger `grub2-mkconfig`. And Prism/MIC will also run it to handle the `extraCommandLine`.